fastlane_bot/events/exchanges/solidly_v2/xfai_v0.py

```diff
@@ -12,6 +12,9 @@ class XFaiV2(SolidlyV2):
     def get_abi(self):
         return XFAI_V0_POOL_ABI
 
+    def get_core_abi(self):
```

Comment:
No need for this function, just place `XFAI_V0_POOL_ABI` directly where you call it below.